opencti-platform/opencti-front/src/private/components/data/connectors/ConnectorWorks.tsx

```diff
@@ -0,0 +1,360 @@
+import React, { FunctionComponent, useEffect, useState } from 'react';
+import { graphql, createRefetchContainer, RelayRefetchProp } from 'react-relay';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import LinearProgress from '@mui/material/LinearProgress';
+import Paper from '@mui/material/Paper';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogActions from '@mui/material/DialogActions';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import { interval } from 'rxjs';
+import { Delete } from 'mdi-material-ui';
+import makeStyles from '@mui/styles/makeStyles';
+import { ConnectorWorks_data$data } from '@components/data/connectors/__generated__/ConnectorWorks_data.graphql';
+import TaskStatus from '../../../../components/TaskStatus';
+import { useFormatter } from '../../../../components/i18n';
+import { FIVE_SECONDS } from '../../../../utils/Time';
+import { commitMutation, MESSAGING$ } from '../../../../relay/environment';
+import Transition from '../../../../components/Transition';
+
+const interval$ = interval(FIVE_SECONDS);
+
+const useStyles = makeStyles(() => ({
+  paper: {
+    height: '100%',
+    minHeight: '100%',
+    margin: '10px 0 20px 0',
+    padding: '15px',
+    borderRadius: 6,
+    position: 'relative',
+  },
+  number: {
+    fontWeight: 600,
+    fontSize: 18,
+  },
+  progress: {
+    borderRadius: 5,
+    height: 10,
+  },
+}));
+
+export const connectorWorksWorkDeletionMutation = graphql`
+  mutation ConnectorWorksWorkDeletionMutation($id: ID!) {
+    workEdit(id: $id) {
+      delete
+    }
+  }
+`;
+
+type WorkMessages = NonNullable<NonNullable<NonNullable<ConnectorWorks_data$data['works']>['edges']>[0]>['node']['errors'];
+
+interface Options {
```

Comment:
Already defined in the code

Reply:
replace by ConnectorWorksQuery$variables